india_forecast_app/models/pvnet/model.py

```diff
@@ -93,6 +94,9 @@ def predict(self, site_id: str, timestamp: dt.datetime):
             for i, batch in enumerate(self.dataloader):
                 log.info(f"Predicting for batch: {i}")
 
+                # save batch
+                save_batch(batch=batch, i=i, model_name=self.name)
```

Comment:
might need to pass site uuid in to this too, as some sites will have the same model name